scripts/iam/README.md

```diff
@@ -0,0 +1,202 @@
+# GenAI IDP IAM Roles
+
+This directory contains IAM role definitions for operating GenAI IDP solution patterns with least-privilege access.
+
+## All Patterns Deployer Role
+
+The `all-patterns-deployer-role.yaml` CloudFormation template creates a least-privilege IAM role for deploying all three GenAI IDP patterns (1, 2, and 3). This role is designed for users who need to deploy and manage patterns but don't have administrator rights.
+
+### Key Features:
+- **Tag-based access control** - Users must be in the deployer group and use required tags
```

Comment:
We need more introduction / context to make this understandable.. we don't know (yet) what it means to be in the 'deployer group' and 'use required tags' so I'm confused already.